spec/asyncapi.md

```diff
@@ -552,8 +479,39 @@ Field Name | Type | Description
 
 This object MAY be extended with [Specification Extensions](#specificationExtensions).
 
+##### Server Variable Object Example
 
+```json
+{
+  "servers": {
+    "production": {
+      "host": "{org}.mycompany.com",
+      "path": "/api/v1",
+      "description": "The production API server.",
+      "protocol": "https",
+      "variables": {
+        "org": {
+          "default": "demo",
+          "description": "This value is assigned by the service provider, in this example `mycompany.com`."
+        }
+      }
+    }
+  }
+}
+```
 
+```yaml
+servers:
+  production:
+    host: '{org}.mycompany.com'
+    path: '/api/v1'
+    description: The production API server.
+    protocol: https
+    variables:
+      org:
+        default: demo
+        description: This value is assigned by the service provider, in this example `mycompany.com`.
+```
```

Comment:
You added here `Server Variable Object Example` but that describes the whole Server Object, here only the:

```yaml
        default: demo
        description: This value is assigned by the service provider, in this example `mycompany.com`.
```

should be written.

Reply:
I think in this case it's justified to have the whole Server Object. Otherwise, there would be no context of where this `demo` variable is coming from.